config/can_config.h

```diff
@@ -46,8 +46,14 @@ typedef enum CANID {
   REPORT_CLAW_MOTION,
   FORCE_SENSOR_VALUE,
 
```

Comment:
Also make sure that allocated CAN IDs are recorded in https://docs.google.com/spreadsheets/d/1-q8Ut22iqccHrx9dzZxglOa9t4iAxWaMrEmVZOrzj-8/edit?usp=sharing

Reply:
And make sure to update references once #114 is merged and the IDs in HW_bridge